src/UI/HTMLTooltip.js

```diff
@@ -261,9 +290,17 @@ export class HTMLTooltip {
         this.transformRuleIndex = null
 
         this.stylesheet = this.shadow.querySelector('link, style')
-        // Un-hide once the style is loaded, to avoid flashing unstyled content
-        this.stylesheet?.addEventListener('load', () =>
-            this.tooltip.removeAttribute('hidden'))
+        // Un-hide once the style and web fonts have loaded, to avoid flashing
+        // unstyled content and layout shifts
+        this.stylesheet?.addEventListener('load', () => {
+            Promise.allSettled([
+                document.fonts.load("normal 13px 'DDG_ProximaNova'"),
+                document.fonts.load("bold 13px 'DDG_ProximaNova'")
+            ]).then(() => {
+                this.tooltip.removeAttribute('hidden')
```

Comment:
We must now also add/remove the `hidden` attribute on the caret (or maybe the parent?), otherwise it may flash on the top-right corner of the screen on slow devices (tested with 6x CPU throttling).

Reply:
@GioSensation Good catch -- and existing issue made more apparent now we're waiting for the web fonts to load. I'll move the hidden attribute to the parent element 👍